baseapp-auth/baseapp_auth/extensions/allauth/headless/tokens/sessions.py

```
@@ -0,0 +1,46 @@
+            try:
+                claim_serializer_class: Callable = import_string(
+                    self.__class__._claim_serializer_class
+                )
+                token = RefreshToken.for_user(user=user)
+                data = claim_serializer_class(user).data
+                for key, value in data.items():
+                    token[key] = value
+                return dict(access_token=dict(refresh=str(token), access=str(token.access_token)))
+            except ImportError:
+                msg = "Could not import serializer '%s'" % self.__class__._claim_serializer_class
+                raise ImportError(msg)
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)